### PR TITLE
Istio 1.4.400

### PR DESCRIPTION
**Pull request by cbron**
Issue: https://github.com/rancher/rancher/issues/25403

Working off the release-v2.3 branch.

Depends on images from linked rancher PR in issue.

### CODE REVIEW

File: charts/rancher-istio/1.4.400/requirements.yaml [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
-    version: 1.4.3
+    version: 1.4.4
     condition: tracing.enabled
   - name: galley
-    version: 1.4.3
+    version: 1.4.4
     condition: galley.enabled
   - name: kiali
-    version: 1.4.3
+    version: 1.9.0
```
Comment (cbron) [inline charts/rancher-istio/1.4.400/requirements.yaml]: Kiali chart doesn't use current istio version

Comment (StrongMonkey) [reply charts/rancher-istio/1.4.400/requirements.yaml]: hmm, this version doesn't mean image version, correct? It means chart version.

Comment (cbron) [reply charts/rancher-istio/1.4.400/requirements.yaml]: Correct, although the image doesn't match either. Its pointing at sub-chart version, which wasn't matching on 1.4.300 either: https://github.com/rancher/system-charts/blob/dev/charts/rancher-istio/1.4.300/charts/kiali/Chart.yaml#L7

Comment (cbron) [reply charts/rancher-istio/1.4.400/requirements.yaml]: reverted to keep in line with upstream

**Review by StrongMonkey — DISMISSED** [review]
LGTM, would prefer this to be split into two PRs, one for rebase and one for istio 1.4.3.

**Comment by cbron** [comment]
Yes, up to @deniseschannon how we should rebase. This was off release-v2.3, but I can cherry pick to wherever.

**Review by deniseschannon — APPROVED** [review]

**Pull request merged**